independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Modifier.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.quarkus.arc.processor;
+
+public class Modifier extends java.lang.reflect.Modifier {
```

Comment:
I don't really think this class is needed. Basically the only use for it is in the tests (it's use in `BeanDeployment` can be replaced with vanilla `Modiier`).
If you absolutely need the class, then I suggest to place it in the tests only

Reply:
Ok, I have moved it into the DI part and refactored it so that's the only place its needed